src/web.rs

```diff
@@ -2,11 +2,14 @@
 
 #![allow(clippy::uninlined_format_args)]
 
+use js_sys::Object;
 use raw_window_handle::WebWindowHandle;
-use wasm_bindgen::JsCast;
+use wasm_bindgen::{JsCast, JsValue};
 use web_sys::CanvasRenderingContext2d;
 use web_sys::HtmlCanvasElement;
 use web_sys::ImageData;
+use web_sys::OffscreenCanvas;
+use web_sys::OffscreenCanvasRenderingContext2d;
```

Comment:
These should be combined into one block.

Reply:
Done.